languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -3750,43 +3750,134 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
-            	<token skip="2" regexp="yes">einen?</token>
-            	<token regexp="yes">einen?</token>
-            	<example>Ich werde nur dieses eine Mal eine Ausnahme machen.</example> <!-- eine (von einen) wird hier als Verb erkannt -->
-            </antipattern>
-            <antipattern>
-				<token>fliegen</token>
-				<token>Fliegen</token>
-				<token>Fliegen</token>
-				<example>Wenn hinter Fliegen Fliegen fliegen, fliegen Fliegen Fliegen nach.</example>
-            </antipattern>
-            <antipattern>
-                <token skip="2" postag='PA2:.+' postag_regexp='yes'></token>
-                <token postag='PA2:.+' postag_regexp='yes'></token>
-                <example>Die bis dahin lose organisierten Vereine organisierten sich nach deutschem Vereinsrecht und strebten Gemeinnützigkeit an.</example>
-            </antipattern>
-            <pattern>
-                <token postag='VER:.+' postag_regexp='yes'></token>
-                <token postag='ART:.+' postag_regexp='yes' min='0'></token>
-                <token postag='SUB:.+' postag_regexp='yes'></token>
-                <token><match no="0"/></token>
-            </pattern>
-            <message>Das Wort "\1" kommt versehentlich doppelt vor oder es fehlt evtl. ein Komma.</message>
-            <suggestion>\1 \2 \3</suggestion>
-            <suggestion>\2 \3 \4</suggestion>
-            <suggestion>\1 \2 \3, \4</suggestion>
-            <short>Doppeltes Verb</short>
-            <example correction="hat Angst|Angst hat|hat Angst, hat">So habe ich mich gefragt, warum Herr Müller <marker>hat Angst hat</marker>, obwohl eine Härtefallregelung für seine Altersgruppe geplant ist.</example>
-            <example>Theodor hat Krebs, hat aber keine Chemo.</example>
-            <example correction="hat Krebs|Krebs hat|hat Krebs, hat">Theodor <marker>hat Krebs hat</marker> aber keine Chemo.</example>
-            <example correction="haben Glück|Glück haben|haben Glück, haben">Wir hoffen, wir <marker>haben Glück haben</marker>.</example>
-            <example>Ich hoffe, er hat Glück gehabt.</example>
-            <example correction="trinken Bier|Bier trinken|trinken Bier, trinken">Wir <marker>trinken Bier trinken</marker> Wein, so soll es sein!</example>
-            <example correction="kaufen Wurst|Wurst kaufen|kaufen Wurst, kaufen">Wenn wir <marker>kaufen Wurst kaufen</marker>, müssen wir sie auch bezahlen.</example>
-            <example correction="tanken Benzin|Benzin tanken|tanken Benzin, tanken">Kann es sein, dass <marker>tanken Benzin tanken</marker> immer teurer wird?</example>
-        </rule>
+        <rulegroup id="DOPPELTES_VERB" name="Doppelt verwendetes Verb" default="temp_off">    <!-- Verb doppelt im Satz, aber nicht direkt hintereinander. -->
+            <rule>        <!-- Hauptsatz -->
```

Comment:
Danke für die Überarbeitung :-) 
Tipp: Regeln in Regelgruppen bitte immer noch durchnummerieren <!--1--> etc., dann kann man auch die Ergebnisse aus dem Diff besser zuordnen.

Reply:
Ja, genau `<!--1-->, <!--2-->` etc. (siehe andere rule groups im Code). Deine Kommentare wie z. B. `<!-- Verb doppelt im Satz, aber nicht direkt hintereinander. -->` aber trotzdem gerne beibehalten, das ist immer sehr hilfreich :-)